services/api-server/src/simcore_service_api_server/api/routes/files.py

```diff
@@ -154,28 +179,47 @@ async def get_file(
             user_id, file_id
         )
         if not stored_files:
-            raise ValueError("Not found in storage")
+            msg = "Not found in storage"
+            raise ValueError(msg)  # noqa: TRY301
 
         stored_file_meta = stored_files[0]
         assert stored_file_meta.file_id  # nosec
 
         # Adapts storage API model to API model
-        file_meta = to_file_api_model(stored_file_meta)
-        return file_meta
+        return to_file_api_model(stored_file_meta)
 
     except (ValueError, ValidationError) as err:
-        logger.debug("File %d not found: %s", file_id, err)
+        _logger.debug("File %d not found: %s", file_id, err)
         raise HTTPException(
             status.HTTP_404_NOT_FOUND,
             detail=f"File with identifier {file_id} not found",
         ) from err
 
 
+@router.delete(
+    "/{file_id}",
+    status_code=status.HTTP_204_NO_CONTENT,
+    responses={**_common_error_responses},
+    include_in_schema=API_SERVER_DEV_FEATURES_ENABLED,
+)
+async def delete_file(
+    file_id: UUID,
+    storage_client: Annotated[StorageApi, Depends(get_api_client(StorageApi))],
+    user_id: Annotated[int, Depends(get_current_user_id)],
+):
+    assert file_id  # nosec
+    assert storage_client  # nsoec
+    assert user_id  # nosec
```

Comment:
why are these asserte nice above they area clearly declared as exiting from the typing.

Reply:
These are temporary to avoid pylint to complain until the implementation is ready.
In any case two of them are used in the msg so I could remove them